docs/HowTo/Interact/API.md

```diff
@@ -0,0 +1,33 @@
+---
+description: How to access APIs
+---
+
+# Access APIs
+
+You can access the [GoQuorum](../../Reference/API-Methods.md) and [Geth](https://geth.ethereum.org/docs/rpc/server) APIs
+using JSON-RPC over HTTP or WebSocket, or GraphQL over HTTP.
+
```

Comment:
Maybe just mention that you need to use the command line options to enable these?